README.md

```diff
@@ -1 +1,207 @@
-Placeholder README for migration of personal repos.
+# The Storytelling Tool
+
+## Social Justice Repair Kit
+
+<https://sojustrepairit.org/>
+The goal of the SJRK is to help youth movements and social justice initiatives to become welcoming environments for
+youth with learning differences, and benefit from the advantages of inclusive design. The resources available in the
+SJRK are intended to be lightweight and easy to deploy, share, repurpose and reuse. The kit will also be openly
+available to any group or individual hosting youth movements, youth action events and social justice movements.
+
+## The Storytelling Tool
+
+More information on this project is available at:
+<https://wiki.fluidproject.org/display/fluid/Web+Storytelling+Tools>
+
+This repository represents the Storytelling Tool, a project which anyone can use to share their story with the world in
+a way that is inclusive, multimodal and accessible. The project primarily uses [Fluid Infusion](https://fluidproject.org/infusion.html),
+so it is assumed that anyone working on it will have some familiarity with Infusion's syntax and philosophy.
+
+This project also uses [Node.js](https://nodejs.org), [Kettle](https://github.com/fluid-project/kettle) and [CouchDB](http://couchdb.apache.org/)
+to build the web-hosting environment that drives the tool.
+
+## How do I use it?
+
+### Running the site
+
+* `npm install` to install dependencies
+* Run a CouchDB server on `localhost:5984`
+  * Docker is an easy way to do this, we recommend the `apache/couchdb` image. To use that image via Docker, run this
+    command: `docker run -p 5984:5984 -d apache/couchdb`
+* Run `node .\src\server\db\dbSetup.js` to configure necessary CouchDB databases (this will also ensure your CouchDB
+  instance is set up in [single-node mode](https://docs.couchdb.org/en/stable/setup/single-node.html))
+  * If you have an [admin user](https://docs.couchdb.org/en/stable/intro/security.html) configured on you CouchDB
+  instance, you will need to provide the credentials to `dbSetup.js`. This can be done by setting the `COUCHDB_URL`
+  environment variable. `dbSetup.js` will use the URL specified in `COUCHDB_URL` when connecting to CouchDB. For
+  example: `COUCHDB_URL=http://admin:adminpassword@localhost:5984`
+* Create a `sjrk.storyTelling.server.config.json5` file in the style of `sjrk.storyTelling.server.config.json5.example`
```

Comment:
Be a little more explicit with this instruction and also the one about "secrets" that this can be done by simply copying the existing files - provide the cp command line to eliminate all doubt in the reader's mind!